CHANGELOG.md

```diff
@@ -13,8 +11,6 @@
 
 **Fixes**:
 
-- `prqlc` no longer displays a prompt when piping a query into its stdin
-  (@max-sixty, #3248).
```

Comment:
Perhaps we need to separate prqlc changes from prql-compiler changes, I thought prqlc changes should be placed in integration.